spec/features/applications/index_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+RSpec.describe "application index page" do
+
+  before(:each) do
+    @app1 = Application.create!(name: "Sam", address: "106 Chatsworth Circle", city: "Sugar Land", state: "TX", zipcode: "77456",description: "Needs a loyal friend", status: "In Progress")
+    @app2 = Application.create!(name: "Poca", address: "7021 Garland", city: "Richmond", state: "TX", zipcode: "77407",description: "Needs a loyal friend", status: "In Progress")
+    @app3 = Application.create!(name: "Prince", address: "4719 Rose Ambers Court", city: "Fresno", state: "TX", zipcode: "77545",description: "Wonderful companion on boring days", status: "Accepted")
+    @app4 = Application.create!(name: "Taj", address: "22655 Briar Forest Drive", city: "Houston", state: "TX", zipcode: "77077",description: "Wonderful companion on boring days", status: "In Progress")
+    
+    @s1 = Shelter.create!(foster_program: true, name: "Dogtopia", city: "Houston", rank: 2)
+    
+    @pet1 = Pet.create!(name: "Buddy", adoptable: true, age: 5, breed: "Shiba", shelter_id: @s1.id)
+    @pet2 = Pet.create!(name: "Jackie", adoptable: false, age: 1, breed: "Inu", shelter_id: @s1.id)
+    @pet_app1 = PetApplication.create!(application: @app1, pet: @pet2)
+    @pet_app2 = PetApplication.create!(application: @app2, pet: @pet1)
+  end 
+end 
```

Comment:
There aren't any actual assertions in this spec file. For the US you have commented below, in this spec you want to test that there is a link to "Start an Application" and that clicking on it takes you to the new app page.

```ruby
expect(page).to have_link("Start an Application")

click_link "Start an Application"

expect(current_path).to eq("/applications/new")
```

From here you can pick up the rest of the test in: `spec/applications/new_spec.rb`